bin/postinstall

```diff
@@ -40,6 +40,18 @@ ${CLI} config:set SECRET_TOKEN="$secret_token"
 # migrate
 ${CLI} run rake ${rake_commands} || true
 
+# Avoid errors on versions < 5.0
+if [ -e "${APP_HOME}/lib/tasks/scm.rb" ]; then
+	# migrate previous repositories with reposman to managed
+	SVN_REPOSITORIES = $(${CLI} config:get SVN_REPOSITORIES)
```

Comment:
`config:get` exits with status 1 when the value does  not exist, so I would use:
```
SVN_REPOSITORIES = $(${CLI} config:get SVN_REPOSITORIES || echo "")
```